.github/workflows/validate.yml

```diff
@@ -10,19 +10,17 @@ on:
     paths:
       - 'api/**'
       - 'config/crd/**'
-    paths-ignore:
-      - 'docs/**'
-      - '**.md'
-      - 'sec-scanners-config.yaml'
+      - '!docs/**'
+      - '!**.md'
+      - '!sec-scanners-config.yaml'
   pull_request:
     branches: [ "main" ]
     paths:
       - 'api/**'
       - 'config/crd/**'
-    paths-ignore:
-      - 'docs/**'
-      - '**.md'
-      - 'sec-scanners-config.yaml'
+      - '!docs/**'
+      - '!**.md'
+      - '!sec-scanners-config.yaml'
```

Comment:
I think for this test it is sufficient to only have path define and to not define anything that should not be scanned.
```suggestion
```